pypeit/spectrographs/keck_nirspec_high.py

```diff
@@ -0,0 +1,1434 @@
+
+        The list of raw data FITS keywords should be those used to populate
+        the :meth:`~pypeit.spectrographs.spectrograph.Spectrograph.configuration_keys`
+        or are used in :meth:`~pypeit.spectrographs.spectrograph.Spectrograph.config_specific_par`
+        for a particular spectrograph, if different from the name of the
+        PypeIt metadata keyword.
+
+        This list is used by :meth:`~pypeit.spectrographs.spectrograph.Spectrograph.subheader_for_spec`
+        to include additional FITS keywords in downstream output files.
+
+        Returns:
+            :obj:`list`: List of keywords from the raw data files that should
+            be propagated in output files.
+        """
+        return ['SCIFILT1', 'SCIFILT2', 'ECHLPOS', 'DISPPOS', 'SLITNAME']
+
+    def pypeit_file_keys(self):
+        """
+        Define the list of keys to be output into a standard PypeIt file.
+
+        Returns:
+            :obj:`list`: The list of keywords in the relevant
+            :class:`~pypeit.metadata.PypeItMetaData` instance to print to the
+            :ref:`pypeit_file`.
+        """
+        pypeit_keys = super().pypeit_file_keys()
+        # TODO: Why are these added here? See
+        # pypeit.metadata.PypeItMetaData.set_pypeit_cols
+        pypeit_keys += ['comb_id', 'bkg_id', 'shift']
+        return pypeit_keys
+
+    def get_echelle_angle_files(self, lamps_list, filter1, filter2):
+        """ Pass back the files required
+        to run the echelle method of wavecalib
+
+        Returns:
+            list: List of files
+        """
+        if filter1 == 'Kband-new' or filter2 == '':
+            band = filter1
+        else:
+            band = filter2
+        
+
+        print(lamps_list, 'Xe' in lamps_list[0])
+        print('filter1 = ', filter1)
+        if 'Xe' in lamps_list[0]:
+            #print('Using Arclamps probably')
+            #print(f'filter1 = {filter1}')
+            if band == 'NIRSPEC-1':
+                angle_fits_file = 'keck_nirspec_y_angle_fits.fits'
+                composite_arc_file = 'keck_nirspec_y_composite_arc.fits'
+            if band == 'NIRSPEC-3':
+                angle_fits_file = 'keck_nirspec_j_angle_fits.fits'
+                composite_arc_file = 'keck_nirspec_j_composite_arc.fits'
+            if band == 'Kband-new':
+                angle_fits_file = 'keck_nirspec_k_angle_fits.fits'
+                composite_arc_file = 'keck_nirspec_k_composite_arc.fits'
+            if band == 'NIRSPEC-7':
+                angle_fits_file = 'keck_nirspec_k_old_angle_fits.fits'
+                composite_arc_file = 'keck_nirspec_k_old_composite_arc.fits'
+        elif 'OH' in lamps_list[0]:
+            print('Using OH Lines')
+            if band == 'NIRSPEC-1':
+                angle_fits_file = 'keck_nirspec_y_OH_angle_fits.fits'
+                composite_arc_file = 'keck_nirspec_y_composite_OH.fits'
+
+        return [angle_fits_file, composite_arc_file]
+
+    '''
```

Comment:
Don't use docstring syntax for comments, here and below.  I would remove all these functions.